org.gnome.Raider.json

```
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.gnome.Raider",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
```

```suggestion
    "runtime-version" : "41",
```

3.28 is more than EOL.